skynet/modules/ttt/summaries/app.py

```diff
@@ -25,13 +27,17 @@ async def app_startup():
 
 
 async def executor_startup():
+    setup_credentials()
+    log.info('Credentials set. Watching for changes...')
```

Comment:
I think we should log this from the credentials thing, since it might not actually be used.